index.html

```diff
@@ -1,248 +1,308 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>My Portfolio</title>
+    <!-- External Css -->
     <link rel="stylesheet" href="assets/css/style.css">
-    <!-- Poppins font form Google APIs cdn -->
-    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
 </head>
+
 <body>
-    <header id="header">
-        <a href="#" id="my_logo">My Logo</a> 
+  <div class="wrap">
+    <header class="toolbar">
+      <div class="container">
+        <a href="#"> My Logo </a>
         <nav>
-            <a href="#" id="img_link">
-                <img src="assets/images/menu.png" id="menu_img" alt="menu">
-            </a>
-        </nav>                
+          <a href="#">
+            <img src="assets/images/menu.png" alt="" />
+          </a>
+          <ul>
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
```

Comment:
- [ ] Kindly add responsiveness to these navlink , so that they may bring the user to the corresponding section once click, it will increase the user experience :+1: 

```bash
<li><a href="#section-id">Portfolio</a></li>
```

![Screenshot from 2023-01-21 12-37-01](https://user-images.githubusercontent.com/80458199/213863696-ddf30098-3132-42d7-800c-90287670e52c.png)